test/paypro.js

```diff
@@ -56,6 +56,7 @@ describe('paypro', function() {
   });
 
   it('Make a PP request with browser', function(done) {
+    xhr.status=200;
```

Comment:
Needs a space either side of the = operator.

Reply:
Done